floor_generator/lib/value_object/embedded.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:analyzer/dart/element/element.dart';
+import 'package:collection/collection.dart';
+import 'package:floor_generator/value_object/field.dart';
+
+class Embedded {
+  final FieldElement fieldElement;
+  final List<Field> fields;
+
+  ClassElement get classElement => fieldElement.type.element as ClassElement;
```

Comment:
💡This can be moved to the constructor so it's only being initialized once instead of on every access.